### PR TITLE
Skip dynamic component generation if the version is avaiable statically

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR adds Skipping dynamic component generation if the version is avaiable statically (with an optional env variable to force it).
The name of the directories are changed to strictly adhere to version name.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -171,12 +172,18 @@ func registerDynamicCapabilities(port string, log logger.Handler) {
...
-	release, err := config.GetLatestReleases(1)
-	if err != nil {
-		log.Info("Could not get latest stable release")
+	// release, err := config.GetLatestReleases(1)
+	// if err != nil {
+	// 	log.Info("Could not get latest stable release")
+	// 	return
+	// }
+	// version := release[0].TagName
+	version := "1.11.4"
```
Comment (utkarsh-pro) [inline main.go:181]: Why is this hardcoded?

Comment (Revolyssup) [reply main.go:181]: Ooops, was testing something and it got committed by mistake, changing it

**Comment by leecalcote** [comment]
Check still failing...

**Comment by Revolyssup** [comment]
> Check still failing...

There is an open issue https://github.com/meshery/meshery-istio/issues/278 , due to which all PR's are failing tests

File: istio/install.go [hunk 2]
```diff
@@ -406,7 +406,11 @@ func unzip(location string, zippedContent io.Reader) error {
 			if err != nil {
 				return ErrUnzipFile(err)
 			}
+<<<<<<< HEAD
+			//#nosec
+=======
 			/* #nosec G307 */
+>>>>>>> upstream/master
```
Comment (leecalcote) [inline istio/install.go:413]: @Revolyssup this is probably an accident.

**Comment by leecalcote** [comment]
Running build jobs

File: istio/oam/register.go [hunk 3]
```diff
@@ -117,3 +118,11 @@ func load(basePath string) ([]schemaDefinitionPathSet, error) {
 
 	return res, nil
 }
+func init() {
+	var err error
+	pathSets, err = load(workloadPath)
+	if err != nil {
+		fmt.Printf("Could not load definitions and schemas for static component registeration: %v", err.Error())
```
Comment (leecalcote) [inline istio/oam/register.go:125]: Can this be a MeshKit error?

Comment (Revolyssup) [reply istio/oam/register.go:125]: load returns a filepath type error, so I didn't use meshkit logger. And this package didn't have the errors.go already, so I didn't create a meshkit error. Will do now

Comment (Revolyssup) [reply istio/oam/register.go:125]: Also this happens before main function is run, so we dont have the meshkit logger instance yet. I can use normal logger for meshkit error, but then it is of no use

**Pull request merged**